### DISCUSSION TITLE
How can I add Auto Typing Text Effect in my profile readme

### DISCUSSION THREAD

**Discussion by LoftyDroid** [community/community#38406] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not sure if it's possible to create typing text effect with markdown so, I try html with CSS but still not working.

I appreciate it if someone can share the code here.

**Comment by DSDmark** [comment, marked as answer]
HI @shaikhahmed243,

use this :-
https://readme-typing-svg.demolab.com/demo/

and you could try this if you like:-
https://typograssy.deno.dev/

**Reply by LoftyDroid** [reply]
Thank you!

**Reply by Soheil-Sohrabi** [reply]
thank you!

**Comment by Redeem-Grimm-Satoshi** [comment]
Great
